sysconfig/bb5/users/packages.yaml

```diff
@@ -1,188 +1,287 @@
-        version: [develop]
-    tk:
-        paths:
-            tk@8.5.13: /usr
-        version: [8.5.13]
-    xz:
-        paths:
-            xz@5.2: /usr
-        version: [5.2]
-    zlib:
-        paths:
-            zlib@1.2.7+shared~static: /usr
-        version: [1.2.7]
-    all:
-        compiler: [intel, gcc@6.4.0]
-        providers:
-            mpi: [hpe-mpi, intel-mpi]
-            scalapack: [intel-mkl, netlib-scalapack]
+  autoconf:
+    paths:
+      autoconf@2.69: /usr
+    version: ['2.69', '2.63']
+  automake:
+    paths:
+      automake@1.13.4: /usr
+    version: [1.13.4]
+  binutils:
+    paths:
+      binutils@2.29.1: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/2018-09-28/install/linux-rhel7-x86_64/gcc-6.4.0/binutils-2.29.1-quhkkp
+  bison:
+    paths:
+      bison@3.0.4: /usr
+    version: [3.0.4]
+  boost:
+    paths:
+      boost@1.67.0: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/18-06-2018/install/linux-rhel7-x86_64/gcc-6.4.0/boost-1.67.0-6skhmz
+      boost@1.67.0 ~graph: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/18-06-2018/install/linux-rhel7-x86_64/gcc-6.4.0/boost-1.67.0-6skhmz
+    version: [1.67.0]
+  bzip2:
+    paths:
+      bzip2@1.0.6: /usr
+    version: [1.0.6]
+  cairo:
+    paths:
+      cairo@1.8.10: /usr
+    version: [1.8.10]
+  cmake:
+    paths:
+      cmake@3.12.0: /gpfs/bbp.cscs.ch/apps/tools/install/linux-rhel7-x86_64/gcc-4.8.5/cmake-3.12.0-ybmxsc
+    version: [3.12.0]
+  cuda:
+    modules:
+      cuda@9.0.176: cuda/9.0.176
+    version: [9.0.176]
+  curl:
+    paths:
+      curl@7.29.0: /usr
+    version: [7.29.0]
+  flatbuffers:
+    paths:
+      flatbuffers@1.8.0 build_type=Release: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/2018-09-28/install/linux-rhel7-x86_64/gcc-6.4.0/flatbuffers-1.8.0-2nsezd
+  flex:
+    paths:
+      flex@2.5.37: /usr
+    version: [2.5.37]
+  fontconfig:
+    paths:
+      fontconfig@2.10.95: /usr
+    version: [2.10.95]
+  freetype:
+    paths:
+      freetype@2.7.1: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/2018-09-28/install/linux-rhel7-x86_64/gcc-6.4.0/freetype-2.7.1-lmnb7p
+  gdbm:
+    paths:
+      gdbm@1.14.1: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/2018-09-28/install/linux-rhel7-x86_64/gcc-6.4.0/gdbm-1.14.1-553gov
+  gettext:
+    paths:
+      gettext@0.19.8.1: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/2018-09-28/install/linux-rhel7-x86_64/gcc-6.4.0/gettext-0.19.8.1-zheggu
+  glib:
+    paths:
+      glib@2.50.3: /usr
+    version: [2.50.3]
+  gtkplus:
+    paths:
+      gtkplus@2.24.31: /usr
+    version: [2.24.31]
+  # hdf5:
+  #   paths:
+  #     hdf5@1.8.12~mpi~hl: /usr
+  #     hdf5@1.10.2+hl+mpi+shared+szip: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/18-06-2018/install/linux-rhel7-x86_64/gcc-6.4.0/hdf5-1.10.2-dueuws
+  #     hdf5@1.10.2 +hl+szip: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/18-06-2018/install/linux-rhel7-x86_64/gcc-6.4.0/hdf5-1.10.2-dueuws
+  #   version: [1.8.12, 1.10.2]
```

Comment:
having hdf5 is necessary....what you suggest here? I can think of providing +mpi and ~mpi installations.

Reply:
Yes, having both by default would be good.

Reply:
I think this failed for me for some weird reason, and I decided that a manual rebuild would be faster. Maybe swapping the version preferences should be enough?